.github/workflows/config.yml

```diff
@@ -8,6 +8,18 @@ on:
   workflow_dispatch:
 
 jobs:
+  clean-files:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
@afontani I think you need a checkout step to begin with...

Reply:
I wonder if you should just have this happen in the `update-results` job. Otherwise, you'd need to upload as artifact, then download as artifact, etc...

Reply:
@joseph-robertson and I decided to change options_lookup upfront and then download it and run the other tests with the formatted file. Then commit it during the `update-results` job.